core/scss/components/checkbox/module.scss

```diff
@@ -30,3 +30,40 @@
 // Component Specific
 @import 'mixins';
 @import 'checkbox';
+
+
+.#{$input__class}-container {
```

Comment:
was there also indentation?

Reply:
@yananym none that I can see in the library